core/src/main/java/hudson/util/RunList.java

```diff
@@ -208,33 +216,9 @@ public R getLastBuild() {
      * @since 2.279
      */
     public RunList<R> filter(Predicate<R> predicate) {
-        return filter(new PredicateAdapter(predicate));
-    }
-
-    private static class PredicateAdapter<T> implements com.google.common.base.Predicate<T> {
-        private final Predicate<T> predicate;
-
-        public PredicateAdapter(Predicate<T> predicate) {
-            this.predicate = predicate;
-        }
-
-        @Override
-        public boolean apply(@Nullable T r) {
-            return predicate.test(r);
-        }
-    }
-
-    /**
-     * Returns elements that satisfy the given predicate.
-     * <em>Warning:</em> this method mutates the original list and then returns it.
-     * @since 1.544
-     * @deprecated use {@link #filter(Predicate)}
-     */
-    @Deprecated
-    public RunList<R> filter(com.google.common.base.Predicate<R> predicate) {
-        size = null;
-        first = null;
-        base = Iterables.filter(base,predicate);
+        base = StreamSupport.stream(base.spliterator(), false).
+            filter(predicate).
+            collect( Collectors.toList());
         return this;
     }
```

Comment:
Unfortunately there is still a usage of this deprecated method; see jan-molak/jenkins-build-monitor-plugin#472. So before we can delete this deprecated method from Jenkins core, we need to update the baseline in Build Monitor View (to gain access to the non-deprecated replacement) and then land jan-molak/jenkins-build-monitor-plugin#472. In the meantime the deprecated method needs to remain in order to not break that plugin.

Reply:
doh not even a plugin hosted here...

Reply:
Hosted "here" enough: https://plugins.jenkins.io/build-monitor-plugin/